### PR TITLE
in-place porting to e2e tests to Puppeteer

### PR DESCRIPTION
**Pull request by ghost**
#### Changes proposed in this Pull Request

* in place update for login test scenario.

#### Testing instructions

running-tests.md has new instructions at the bottom on how to run.

*

Purpose:
Porting e2e tests to Puppeteer from WebDriver.io

### CODE REVIEW

File: test/e2e/lib/async-base-container.js [hunk 1]
```diff
@@ -33,6 +33,16 @@ export default class AsyncBaseContainer {
...
+		if ( driverManager.usePuppeteer() === 'true' ) {
+			const page = await driver.newPage();
+			page.goto( url, { waitUntil: 'networkidle2' } );
+
+			if ( ! page.url ) {
+				throw new Error( `URL is required to visit the ${ page.name }` );
+			}
+			return page;
+		}
+
```
Comment (bsessions85) [inline test/e2e/lib/async-base-container.js:45]: This will bypass `_expectInit()` which runs some checks for a/b test config and checks for console errors

**Review by bsessions85 — COMMENTED** [review]
This looks good. I noticed one potential issue with the `visit()` and I would be curious to hear your thoughts on how we might be able to make these updates one by one without having very ugly diffs like this one for `wp-log-in-out-spec.js`. Putting in the if statement made it think every line was changed and really you just added a small chunk

**Comment by sixhours** [comment]
Looks like this contributor's account is no longer active. @bsessions85 is this something you still think should happen or should we close it?

**Comment by bsessions85** [comment]
@sixhours  We can close this one.  Thanks!

**Pull request closed without merging**